### PR TITLE
Add previews support to LinkTo

### PR DESCRIPTION
**Pull request by VassilIordanov**
remains to be done:
- [ ] improve interaction and modal position
- [ ] make Show components more responsive
- [ ] refactor code to make Show pages cleaner (possibly make them simpler components and not pages)

#### User changes
- On mouse hover on, most links now render a preview of the object they link to

- [ ] anet.yml needs change
- [ ] db needs migration
- [ ] documentation has changed
- [ ] graphql schema has changed

### Checklist
  - [ ] Described the user behavior in PR body
  - [ ] Referenced/updated all related issues
  - [ ] commits follow a `repo#issue: Title` title format and [these 7 rules](https://chris.beams.io/posts/git-commit/)
  - [ ] commits have a [clean history](https://epage.github.io/dev/commits/), otherwise PR may be squash-merged
  - [ ] Added and/or updated unit tests
  - [ ] Added and/or updated e2e tests
  - [ ] Added and/or updated data migrations
  - [ ] Updated documentation
  - [ ] Resolved all build errors and warnings
  - [ ] Opened debt issues for anything not resolved here

### CODE REVIEW

File: client/src/components/LinkTo.js [hunk 1]
```diff
@@ -5,6 +5,24 @@ import * as Models from "models"
...
+
+export const MODEL_TO_COMPONENT = {
+  [Models.AuthorizationGroup]: AuthorizationGroupShow,
+  [Models.Location]: LocationShow,
+  [Models.Organization]: OrganizationShow,
+  [Models.Person]: PersonShow,
+  [Models.Position]: PositionShow,
+  [Models.Report]: ReportShow,
+  [Models.Task]: TaskShow
+}
```
Comment (gjvoosten) [inline client/src/components/LinkTo.js:25]: Is it possible to use a specific preview component here (`TaskPreview` etc.), extract the specific part you want to see as part of the preview from the Show page into it, and include that extracted Preview component in the Show page?

**Review by gjvoosten — CHANGES_REQUESTED** [review]

File: client/src/components/LinkTo.js (hunk 1 above)
Comment (cemalettin-simsoft) [reply client/src/components/LinkTo.js:25]: Yes, adds a little bit of repeating but we can work with it much easier. Query less data etc...

**Comment by cemalettin-simsoft** [comment]
I was thinking something like this. What do you think? @VassilIordanov @gjvoosten

**Comment by cemalettin-simsoft** [comment]
I will start from zero and create a new branch

**Pull request closed without merging**